keel-core-test/src/main/kotlin/com/netflix/spinnaker/keel/persistence/UnhappyVetoRepositoryTests.kt

```diff
@@ -132,3 +157,9 @@ abstract class UnhappyVetoRepositoryTests<T : UnhappyVetoRepository> : JUnit5Min
     }
   }
 }
+
+private val Assertion.Builder<UnhappyVetoStatus>.shouldSkip: Assertion.Builder<Boolean>
+  get() = get("should skip") { shouldSkip }
+
+private val Assertion.Builder<UnhappyVetoStatus>.shouldRecheck: Assertion.Builder<Boolean>
+  get() = get("should recheck") { shouldRecheck }
```

Comment:
I was just finding it impossible to tell which field was which when the test was failing. This adds a description to the output.